src/test-data/CommonData.js

```diff
@@ -123,6 +123,12 @@ export const metadataTestData = {
   roles: exampleRoles,
   locations: exampleLocations,
   version: "Revision 5",
+  props: [
+    {
+      name: "keywords",
+      value: "Keyword_1, Keyword_2,Keyword_3",
+    },
+  ],
 };
```

Comment:
Please also have tests for:

```json
{"props": []}
```

Displays nothing

```json
{
  "props": [
    {
      "name": "keywords",
      "value": "",
    }
  ]
}
```

Displays nothing

```json
{
  "props": [
    {
      "name": "keywords",
      "value": "Keyword_1",
    }
  ]
}
```

Displays 1 keyword

```json
{
  "props": [
    {
      "name": "keywords",
      "value": "Keyword_1",
      "ns": "https://test.easydynamics.com/ns/oscal"
    }
  ]
}
```

Displays nothing